### PR TITLE
Support torch.mean with dim=None

### PR DESCRIPTION
**Pull request by wonjoolee95**
This fixes https://github.com/pytorch/pytorch/pull/81286

### CODE REVIEW

**Comment by wonjoolee95** [comment]
Build succeeded locally

**Review by JackCaoG — APPROVED** [review]
Thanks!
